docs/backend/cookbook/new_channel.md

```diff
@@ -195,6 +194,52 @@ class ProcessExportCommandHandler
 }
 ```
 
+## Export
+
+It is entity that reflects one export process. One channel can have many exports.
+
+Export lives under `Ergonode\Channel\Domain\Entity` namespace.
+
+It stores following data:
+
+* when process was started and finished,
+* what is the status of that process,
+* to which Channel it belongs,
+
+It has methods for:
+* starting export process
+* stopping export process
+* ending export process
+
+### Export Lines
+
+Export lines are identified by ExportLineId (`Ergonode\Channel\Domain\ValueObject\ExportLineId`)
+
+Data for Export Lines are store in ExportLineData data structure (`Ergonode\ExporterFile\Infrastructure\DataStructure\ExportLineData`)
```

Comment:
This is a description of a specific implementation of. In another module is not present.

Reply:
Maybe add info
`For example ...`